recipes/visp/bld.bat

```diff
@@ -0,0 +1,34 @@
+setlocal EnableDelayedExpansion
+
+mkdir build
+cd build
+
+set "CC=clang-cl.exe"
+set "CXX=clang-cl.exe"
+set "CL=/MP"
+
+::Configure
+cmake ^
+    %SRC_DIR% ^
+    -G Ninja ^
+    -DOGRE_DIR="%LIBRARY_PREFIX%\cmake" ^
```

Comment:
Could you please:
 - define a variable for `OGRE_DIR` similarly to what's done for Unix
 - document the reason for this change, referring to the patch

?